helm/harbor-operator-app/charts/harbor-operator/charts/postgres-operator/templates/clusterrole-postgres-pod.yaml

```diff
@@ -63,6 +63,15 @@ rules:
   - services
   verbs:
   - create
+# to let postgres pod create emptyDir and configMap volume
```

Comment:
This is in the upstream chart, right? This would cause issues when we update from upstream. Is it possible to just put these files into `templates`? That way we have full control over it.

Reply:
I have moved them over to templates folder